en/sql/function/json_fn.rst

```diff
@@ -477,3 +477,115 @@ The following json path will match all json paths that end with a json array ind
+::
+
+      json_contains_path('[{"0":0},1,"2",{"three":3}]', 'all', '$[0]', '$[0]."0"', '$[1]', '$[2]', '$[3]')
+    ======================================================================================================
+                                                                                                         1
+
+.. code-block:: sql
+
+    SELECT JSON_CONTAINS_PATH ('[{"0":0},1,"2",{"three":3}]', 'all', '$[0]', '$[0]."0"', '$[1]', '$[2]', '$[3]', '$.inexistent');
+::
+
+      json_contains_path('[{"0":0},1,"2",{"three":3}]', 'all', '$[0]', '$[0]."0"', '$[1]', '$[2]', '$[3]', '$.inexistent')
+    ======================================================================================================================
+                                                                                                                         0
+
+The JSON_CONTAINS_PATH function supports wildcards inside json paths.
+
+.. code-block:: sql
+
+    SELECT JSON_CONTAINS_PATH ('[{"0":0},1,"2",{"three":3}]', 'one', '$.inexistent', '$[*]."three"');
+::
+
+      json_contains_path('[{"0":0},1,"2",{"three":3}]', 'one', '$[*]."three"')
+    ==========================================================================
+                                                                             1
+
+JSON_CONTAINS
+===================================
+
+.. function:: JSON_CONTAINS (json_doc doc1, json_doc doc2 [, json path])
+
+  The **JSON_CONTAINS** function verifies whether the doc2 is contained inside the doc1 at the optionally specified path.
+  A json element contains another json element if the following recursive rules are satisfied:
+- A json scalar contains another json scalar if they have the same type (their JSON_TYPE () are equal) and are equal. As an exception, json integer can be compared and equal to json double (even if their JSON_TYPE () evaluation are different).
+- A json array contains a json scalar or a json object if any of json array's elements contains the json_nonarray.
+- A json array contains another json array if all the second json array's elements are contained in the first json array.
+- A json object contains another json object if, for every (key2, value2) pair in the second object, there exists a (key1, value1) pair in the first object with key1=key2 and value2 contained in value1.
+  Otherwise the json element is not contained.
+
+  Returns whether doc2 is contained in root json element of doc1 if no json path argument is given.
+  Returns NULL if any argument is NULL.
+  An error occurs if any argument is invalid.
+
+.. code-block:: sql
+
+    SELECT JSON_CONTAINS ('"simple"','"simple"');
+::
+
+      json_contains('"simple"', '"simple"')
+    =======================================
+                                          1
+
+.. code-block:: sql
+
+    SELECT JSON_CONTAINS ('["not important", "important"]','"important"');
```

Comment:
should be more readable with names shorter than "importantx"; some examples can get very long.